sjsonnet/src/sjsonnet/Util.scala

```diff
@@ -0,0 +1,19 @@
+package sjsonnet
+
+
+object Util{
+  def binarySearch(lineStartsMin: Int, lineStartsMax: Int, lineStarts: Array[Int], index: Int): Int = {
```

Comment:
I think `Arrays.binarySearch` would work as well.

Reply:
The issue with `Arrays.binarySearch` is that it only can check for exact `value == input`, whereas this binary search needs to check for `value > input`. In theory it should be about the same logic, but I didn't find any convenient verison built in :/

Reply:
I think it returns the `-insertion_point` for a value that's not in the array, if I understand this correctly:

Returns:
index of the search key, if it is contained in the array within the specified range; otherwise, (-(insertion point) - 1). The insertion point is defined as the point at which the key would be inserted into the array: the index of the first element in the range greater than the key, or toIndex if all elements in the range are less than the specified key. Note that this guarantees that the return value will be >= 0 if and only if the key is found.

Reply:
Ah cool find, I'll make use of that then!

Reply:
Turns out that the comment explaining how we use the result of `java.util.Arrays.binarySearch` is longer than the binary search implementation in the first place. OTOH it lets me delete 115 lines of test suite, so definitely still a win overall